### DISCUSSION TITLE
F-Droid 1.18.0 does not show NewPipe update 0.26.1 on Android 10 ?

### DISCUSSION THREAD

**Discussion by attilalearning** [TeamNewPipe/NewPipe#10947] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

F-Droid 1.18.0 on Android 10 does not show NewPipe 0.26.1 update. See link to F-Droid forum, where they say it is a NewPipe issue? As NewPipe does not 

### Additional information

https://forum.f-droid.org/t/newpipe-update-request/3808/78
https://forum.f-droid.org/t/newpipe-update-request/3808/83
https://forum.f-droid.org/t/newpipe-update-request/3808/88  <----

( https://github.com/TeamNewPipe/NewPipe/issues/10746 )

**Comment by AudricV** [comment]
Read again what you provided, everything is written it it. Duplicate of #10746.
